src/app/shared/services/iam.service.ts

```diff
@@ -237,11 +238,15 @@ export class IamService {
     return from(source).pipe(finalize(() => this.loadingService.hide()));
   }
 
-  private getChainConfig() {
-    const chainConfig: any = {
+  private getChainConfig(): Partial<ChainConfig> {
+    const chainConfig: Partial<ChainConfig> = {
       rpcUrl: this.envService.rpcUrl,
     };
 
+    if (this.envService.claimManagerAddress) {
```

Comment:
Will this.envService be always available or it makes sens to add this.envService?.claimManagerAddress ?

Reply:
It will be always available. This is static js file for configuration purposes. The plan is to build once, deploy everywhere and just change properties in env.js file. This file is loaded when application is initializing. So if this file won't be available, then switchboard will not work.

Reply:
To have better overview: https://www.jvandemo.com/how-to-use-environment-variables-to-configure-your-angular-application-without-a-rebuild/